### DISCUSSION TITLE
Greenfield API: return BTC amount for Invoice

### DISCUSSION THREAD

**Discussion by alexgleason** [btcpayserver/btcpayserver#2254] [Ideas / Feature Requests]
When GET'ing an Invoice through the Greenfield API, it returns only the *converted* amount, eg "$10 USD", but for bookkeeping purposes I would like it to also return the source amount, eg "0.0002695 BTC"

**Comment by Kukks** [comment]
Please see https://docs.btcpayserver.org/API/Greenfield/v1/#operation/Invoices_GetInvoicePaymentMethods
A lot more info is provided in terms of rates per payment method there.
